pyramid_oereb/contrib/data_sources/standard/models/main.py

```diff
@@ -12,17 +12,16 @@
 
 The geographical projection system which is used out of the box is LV95 aka EPSG:2056. Of course you can
 configure a different one.
-The name of the schema will be::
+The name of the schema will be:
 
     pyramid_oereb_main
 
 But you can change it also via configuration.
 
-    .. note::
-
-        Whenever you configure your own sqlalchemy ORM's to use them in this application you must imitate
-        the behaviour of the ORM's here. This means the names class variables as well as the types of these
-        variables.
+Note:
```

Comment:
I this working with automatic doc generation?

Reply:
I am not sure I still need to test this. But I think it should from what I saw how Notes are added.
I'll check this to be sure.

Reply:
Seems to work with the notes:
![Docs_Notes](https://user-images.githubusercontent.com/1460598/155724710-08123d1f-3ec9-42e4-9995-8ff6d5a07323.png)